pkg/volume/glusterfs/glusterfs.go

```diff
@@ -67,6 +68,8 @@ const (
 	gciGlusterMountBinariesPath = "/sbin/mount.glusterfs"
 	defaultGidMin               = 2000
 	defaultGidMax               = math.MaxUint32
+	annDynamicallyProvisioned   = "pv.kubernetes.io/provisioned-by"
+	annStorageClass             = "volume.beta.kubernetes.io/storage-class"
```

Comment:
don't need these, use use `util.GetVolumeStorageClass(pv)` to extract the class name, and I don't think we care who provisioned the pv, only whether it has an associated gid